### ISSUE TITLE
On restart, last used picture is used instead of unicolor of `clear`

### ISSUE THREAD

**Issue by allergicapple** [LGFae/swww#379]
Scenario:
- Set a wallpaper to an image with `swww img cat.jpg`
- Clear the wallpaper with `swww clear 446688`
- Restart service / reboot computer / ...
- Start `swww-daemon`

Excpected:
- Wallpaper is single color sky blue again.

Actual:
- Wallpaper is cat.

I think I misunderstand the `clear` command to mean something like "uniform single-color wallpaper" when it is not because I would expect swww to restore this on restart and not the last picture set with `img`.

Using latest git:
```console
$ swww --version
swww 0.9.5-masterV3
```

Installed from AUR:
```console
$ pacman -Qi swww
Installed From  : None
Name            : swww-git
Version         : 0.9.5.r172.g0db3f4e-1
Description     : Efficient animated wallpaper daemon for Wayland, controlled at runtime
Architecture    : x86_64
URL             : https://github.com/LGFae/swww
[...]
```

**Comment by kerty0** [comment]
Yes, `swww-daemon` restores wallpaper based on cache and `swww img` modifies it but `swww clear` doesn't. You can set persistent color wallpaper with `swww img 0x446688`.

**Comment by allergicapple** [comment]
> Yes, `swww-daemon` restores wallpaper based on cache and `swww img` modifies it but `swww clear` doesn't. You can set persistent color wallpaper with `swww img 0x446688`.

Hm, no, that doesn't work.
- First I was unaware I can replace a filename with `0xrrggbb` as it's both not documented and actually counter-intuitive
- Second, it does not work; the background gets replaced with vertical stripes, so unexpectedly it does _something_, but not a solid color
